engine/table/src/main/java/io/deephaven/engine/util/PythonScopeJpyImpl.java

```diff
@@ -189,7 +189,16 @@ public static Object convert(PyObject pyObject) {
         } else if (pyObject.isCallable()) {
             return wrapCallable(pyObject);
         } else if (pyObject.isConvertible()) {
-            return pyObject.getObjectValue();
+            Object objValue = pyObject.getObjectValue();
+            // workaround a JPY issue with determining the correct Java type a Python 'int' should be
+            // mapped to (int or long) without causing an implicit overflow
+            if (objValue instanceof  Integer || objValue.getClass() == int.class) {
```

Comment:
Do we think the second case will ever trigger? I would expect that the instanceof is sufficient.

Reply:
I am not 100% sure, and that's why I threw it in there just in case there are some edges cases that are not obvious to us now

Reply:
Why don't we use `PyObject.isLong()` or `PyObject.isInt()`?

Reply:
isLong / isInt are the same (python 3), but it won't actually tell you how many bits are in the python int.

Reply:
An alternative would be to do `PyObject.isLong()`, `PyObject.getLongValue()`, and downcast back to `int` if appropriate (or, choose to have `convert` never return an int and always return a long)

Reply:
I like that.

Reply:
@cpwright do you like the above? Preference for always `long`, or `int` if it fits?

Reply:
I would suggest int if it fits.